README.md

```diff
@@ -161,7 +161,14 @@ end
 
 ## Cursors
 
-You can use `Mongoid::Scroll::Cursor.from_record` to generate a cursor. A cursor points at the last record of the previous iteration and unlike MongoDB cursors will not expire.
+You can use `Mongoid::Scroll::Cursor.from_record` to generate a cursor. A cursor points at the last record of the 
+
```

Comment:
Let's get rid of the extra space ;) generally don't wrap markdown either 
https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html